From ba34548e091d5622e9eeae04f5bf3bc87ead5b98 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Mon, 28 May 2018 16:20:13 +0800
Subject: [PATCH 611/666] powerpc: mmu-book3e: fix a typo to avoid build error

Fix a typo to avoid the function book3e_tlb_exists() was not defined.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/powerpc/include/asm/mmu-book3e.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/powerpc/include/asm/mmu-book3e.h b/arch/powerpc/include/asm/mmu-book3e.h
index 0745dc1..fffbf0c 100644
--- a/arch/powerpc/include/asm/mmu-book3e.h
+++ b/arch/powerpc/include/asm/mmu-book3e.h
@@ -319,11 +319,10 @@ static inline void book3e_tlb_lock(void)
 
 static inline void book3e_tlb_unlock(void)
 {}
+#endif
 
 int book3e_tlb_exists(unsigned long ea, unsigned long pid);
 
-#endif
-
 #endif /* !__ASSEMBLY__ */
 
 #endif /* _ASM_POWERPC_MMU_BOOK3E_H_ */
-- 
2.7.4

